rules/id/id-jawi.js

```diff
@@ -0,0 +1,50 @@
+﻿( function ( $ ) {
+	'use strict';
+	var jawiKeyboard = {
+		id: 'id-jawi',
+		name: 'Jawi keyboard',
+		description: 'Jawi keyboard for Indonesian Wikisource',
+		date: '2023-09-21',
+		URL: 'https://www.mediawiki.org/wiki/Help:Extension:UniversalLanguageSelector/Input_methods/jawi-keyboard',
+		author: 'Benny Lin, Pungkas, Michael',
+		license: 'GPLv3',
+		version: '1.0',
+		contextLength: 3,
+		maxKeyLength: 3,
```

Comment:
Is it 3? 
Please check with the definition of field at https://github.com/wikimedia/jquery.ime/tree/master/rules#input-method-definition